sway-lib-std/tests/test_projects/token_ops/mod.rs

```diff
@@ -138,6 +138,70 @@ async fn can_force_transfer() {
     //assert_eq!(balance_result.value, 42);
 }
 
+#[tokio::test]
+async fn can_mint_and_send_to_contract() {
+    let (provider, wallet) = setup_test_provider_and_wallet().await;
+    let (fuelcoin_instance, fuelcoin_id) =
+        get_fuelcoin_instance(provider.clone(), wallet.clone()).await;
+    let balance_id = get_balance_contract_id(provider, wallet).await;
+    let amount = 55u64;
+
+    let asset_id = testfuelcoincontract_mod::ContractId {
```

Comment:
Can we directly use the newly available `ContractId` type? or would this require a new version of the SDK that we can't build with yet?

Reply:
This requires the newer version of the SDK, so I'll be cleaning this up shortly along with all the other tests once I'm able to do so.